packages/kbn-lens-embeddable-utils/attribute_builder/visualization_types/layers/xy_by_value_annotation_layer.ts

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
The `attribute_builder` will be deprecated in the future in favor of the [`config_builder` ](https://github.com/elastic/kibana/tree/main/packages/kbn-lens-embeddable-utils/config_builder) created recently by the visualizations team.

Reply:
@crespocarlos thanks for the heads up. I will create a separate issue to replace `attribute_builder` usage with `config_builder` in observability plugin.

Reply:
created https://github.com/elastic/kibana/issues/176146

Reply:
thanks @benakansara !